test/helpers/cons.go

```diff
@@ -22,7 +22,7 @@ import (
 
 	k8sConst "github.com/cilium/cilium/pkg/k8s/apis/cilium.io"
 	"github.com/cilium/cilium/pkg/versioncheck"
-	"github.com/cilium/cilium/test/ginkgo-ext"
+	ginkgoext "github.com/cilium/cilium/test/ginkgo-ext"
```

Comment:
Is this used somewhere?

Reply:
I think this was automatically added by `goimports` (which my editor runs automatically on save). Can drop it if needed.

Reply:
I see, thanks! Let's assume that go tooling knows what is doing and move on ;)